themes/default/views/administrate/setup/interface_editor/screen_html.php

```diff
@@ -25,33 +25,35 @@
  *
  * ----------------------------------------------------------------------
  */
- 	$t_subject = $this->getVar('t_subject');
```

Comment:
Same comment about fixing formatting goes for these views.  They aren't perfect by any means but since I was changing them anyway I decided to fix a bit of the strangeness inherited from the base theme.  You could possibly argue this will make future merges more difficult unnecessarily but I think git will be okay with it (?)  Also it does indicate that the file has been changed by us, although that isn't an indicator that we've used before.